app/models/discussion.rb

```diff
@@ -47,11 +47,15 @@ def try_solve!
   def navigable_content_in(_)
     nil
   end
-  
+
   def target
     self
   end
 
+  def content_used_in?(organization)
```

Comment:
This was added because testing locally I've realized that immersive redirection for a specific discussion was not working: https://github.com/mumuki/mumuki-laboratory/pull/1609/commits/168a507db3546293f9eecc37b467c67269fceaf8